routes/utils.js

```diff
@@ -57,6 +57,25 @@ utils.ensureNoSlash = function (url) {
     return url;
 };
 
+utils.makePagingUri = function (req, uri, filterFields) {
+    var startIndex = (req.query.pageIndex && req.query.pageSize) ?  (req.query.pageIndex - 1) * req.query.pageSize : 0;
+    uri  = uri+ "&offset="+startIndex;
+    uri  = ( req.query.pageSize )  ? uri + "&limit="+req.query.pageSize : uri;
+    uri  = ( req.query.sortField ) ? uri +"&order_by="+req.query.sortField+"%20"+req.query.sortOrder.toUpperCase() : uri;
```

Comment:
Rather use

const qs = require('querystring'); // might already be present here

... and the `qs.escape(whatever)` method for URI encoding. That's what is used all over the code.

I can also recommend using template strings for string formatting: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings
That mostly makes it easier to read.